script.js

```diff
@@ -242,6 +242,32 @@ otherButtons.forEach((button) => {
 
 }
 
+// Get modal element
+const modal = document.getElementById('myModal');
+
+// Get open modal button
+const openModalBtn = document.getElementById('openModalBtn');
+
+// Get close button
+const closeBtn = document.querySelector('.close');
+
+// Listen for open click
+openModalBtn.addEventListener('click', () => {
+    modal.style.display = 'block'; // Show the modal
+});
+
+// Listen for close click
+closeBtn.addEventListener('click', () => {
+    modal.style.display = 'none'; // Hide the modal
+});
+
+// Listen for outside click
+window.addEventListener('click', (event) => {
```

Comment:
the code feels a bit unclear here, how is it that when they click the modal that it triggers a close but the comment says they clicked outside the modal.

I haven't tested if it works but seems a bit confusing, also I'd consider adding a check to see if the modal is being displayed before running display none here. 

Is not super necessary in a smaller project like this but having thorough checks of conditions before running code prevents alot of bugs in the longterm on bigger projects. More of a good practice. This code would run fine though.

Reply:
I removed this whole event listener, the modal ended up being bigger than i thought it would be, and this is no longer needed.